README.md

```diff
@@ -20,7 +20,7 @@ If you'd like to read it locally:
 $ git clone https://github.com/rust-lang-nursery/rust-cookbook
 $ cd rust-cookbook
 $ cargo install mdbook --vers "0.4.43"
-$ mdbook serve --open
+$ cargo xtask book build
```

Comment:
I think both of these things should be highlighted in the readme, but this included note should have its own section rather than clobbering the mdbook serve --open operation.

Reply:
sure I'll revert the changes and create a separate readme

Reply:
is there a reason why the version of the mdbook in the contribution guidelines is different from the readme?